### ISSUE TITLE
Training data pipeline — conversation export parser

### ISSUE THREAD

**Issue by rangersui** [rangersui/Elastik#23]
Build parser for Claude/ChatGPT conversation exports.

Input: JSON exports from Claude and ChatGPT
Output: standardized JSONL training pairs

Each record:
  {
    "instruction": "user's original message",
    "context": "current world state / what AI read",
    "tool_calls": ["GET /info", "POST /tasks/write"],
    "response": "AI's final action + reply"
  }

Parser must handle:
  - Claude tool_use blocks (MCP http calls)
  - ChatGPT function_call blocks
  - Multi-turn conversations → split into pairs
  - Filter noise (empty messages, system heartbeats)

This is the foundation for fine-tuning the local minister.
Data already exists in conversation exports.
Parser accuracy = training data quality = model quality.

First step: Claude Code analyzes one export file,
maps the structure, writes the parser.

**Comment by rangersui** [comment]
Closing after the V6 boundary reset: conversation export parsing and training-data generation are separate application/data-pipeline work, not core or SDK primitives.
